<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResampleRegularMesh</class>
 <widget class="QDialog" name="ResampleRegularMesh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ResampleRegularMesh</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="dimOuputMeshGroupBox_">
     <property name="title">
      <string>Dimensions of output mesh</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>X-axis</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="yAxisDimensionComboBox_">
          <item>
           <property name="text">
            <string>Number of samples</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Scaling factor</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="xAxisDoubleSpinBox_">
          <property name="maximum">
           <double>10000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="yAxisDoubleSpinBox_"/>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="xAxisDimensionComboBox_">
          <item>
           <property name="text">
            <string>Number of samples</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Scaling factor</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="zAxisDoubleSpinBox_"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Y-axis</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Z-axis</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="zAxisDimensionComboBox_">
          <item>
           <property name="text">
            <string>Number of samples</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Scaling factor</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="resampleKernalGroupBox_">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>112</height>
      </size>
     </property>
     <property name="title">
      <string>Resampling Kernel</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="kernelComboBox_">
        <item>
         <property name="text">
          <string>Box</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tent</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cubic (Catmull-Rom)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cubic (B-Spline)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quartic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gaussian</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="gaussianSigmaLabel_">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Gaussian sigma</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="gaussianSigmaDoubleSpinBox_">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>100000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="gaussianExtendLabel_">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Guassian extend</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="gaussianExtendDoubleSpinBox_">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
